app/components/businesses/admin_component.rb

```diff
@@ -0,0 +1,14 @@
+module Businesses
+  class AdminComponent < ApplicationComponent
+    private attr_reader :business, :user
```

Comment:
`business` is used in the view, so let's make that public and keep `user `private.